.github/workflows/create-reports.yaml

```diff
@@ -4,13 +4,25 @@ name: Automated Evaluation Reports and Projects
 on:
   workflow_dispatch:
 
+permissions:
+  contents: write
+  pull-requests: write
+
 jobs:
   evaluation:
     runs-on: windows-latest
     steps:
     - name: Checkout repository
       uses: actions/checkout@v4
 
+    - name: Rebase onto develop
+      run: |
+        git config --global user.name 'github-actions'
+        git config --global user.email 'github-actions@github.com'
+        git fetch origin develop
+        git rebase -Xours origin/develop
+        git reset HEAD^

```

Comment:
Why do we need `git reset HEAD^` ?

Reply:
After rebase, if I don't reset the current branch, the commits from `create-reports` that includes `tools.csv`, `models.csv`, etc. get included in the PR.
Here is the resulting test when this happens: 

- [Action](https://github.com/pchelle/OSP-PBPK-Model-Library/actions/runs/12913155990/job/36009714419)
- [PR](https://github.com/pchelle/OSP-PBPK-Model-Library/pull/14/files)

Reply:
OK, I see.